tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -45,6 +45,23 @@ limitations under the License.
 #define GPU_AXIS_KERNEL_LOOP(i, n, axis) \
   for (int i : ::tensorflow::GpuGridRange##axis<int>(n))
 
+#if TENSORFLOW_USE_ROCM
+
+#define cub hipcub
+#define cudaSuccess hipSuccess
+#define cudaGetErrorString hipGetErrorString
+#define cudaStream_t hipStream_t
+#define cudaGetLastError hipGetLastError
+#define cudaError hipError
+
+#define CUDA_1D_KERNEL_LOOP(i, n)                                   \
```

Comment:
Please help check if this macro can be abolished and use `GPU_1D_KERNEL_LOOP` also defined in this header file.